### PR TITLE
Propagate tags to app-layers in docker images for all langs

### PR DESCRIPTION
**Pull request by vsbus**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
We are using `py3_image()` with `layers` parameter in one of our projects. These images have `tags = ["manual"]` so we don't expect them to build with `bazel build //...`. However, we observing that `.tar` files with layers are still built. That happened because tags were not propagated to the layer's targets. This PR is trying to fix the issue.

Issue Number: N/A

## What is the new behavior?
`bazel build //...` doesn't build extra targets like layers when manual tag is used.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by vsbus** [comment]
Hey @gravypod , please take another look

**Review by gravypod — APPROVED** [review]
Looks good. Lets just get a second review from @alexeagle since this will be a user-visible change.

**Review by alexeagle — APPROVED** [review]
thanks!

**Pull request merged**
